src/current/_redirects

```diff
@@ -381,12 +381,11 @@ layout: null
 /docs/cockroachcloud/console-access-management.html /docs/cockroachcloud/managing-access.html 301
 /docs/cockroachcloud/connect-to-a-free-cluster.html /docs/cockroachcloud/connect-to-a-serverless-cluster.html 301
 /docs/cockroachcloud/create-a-free-cluster.html /docs/cockroachcloud/create-a-serverless-cluster.html 301
-/docs/cockroachcloud/planning-your-serverless-cluster.html /docs/cockroachcloud/learn-about-pricing.html 301
 /docs/cockroachcloud/migrate-from-free-to-dedicated.html /docs/cockroachcloud/migrate-from-serverless-to-dedicated.html 301
 /docs/cockroachcloud/free-faqs.html /docs/cockroachcloud/serverless-faqs.html 301
 /docs/cockroachcloud/connect-to-a-free-cluster /docs/cockroachcloud/connect-to-a-serverless-cluster.html 301
 /docs/cockroachcloud/create-a-free-cluster /docs/cockroachcloud/create-a-serverless-cluster.html 301
-/docs/cockroachcloud/planning-your-serverless-cluster /docs/cockroachcloud/learn-about-pricing.html 301
+/docs/cockroachcloud/planning-your-serverless-cluster /docs/cockroachcloud/plan-your-cluster-serverless.html 301
```

Comment:
I think I have to redo all this anyway because the redirect system was just updated, but i'm not sure I understand these suggestions. We have to redirect from the same pages with and without the `.html` suffix?

Reply:
Maybe I'm misreading the instructions but I thought that was generated automatically

Reply:
On line 252 of `src/current/_data/redirects.yml`, change the destination from `learn-about-pricing.html` to `plan-your-cluster-serverless.html`.

Reply:
We previously had to manually create both redirects. Nick's change today makes it automatic.